packages/types/src/lo-hi.ts

```diff
@@ -34,9 +34,9 @@ export const splitLoHi = (value: bigint): Required<LoHi> => {
  * returns a bigint that combines these two parts.
  * To achieve this, we shift `hi` 8 bytes to the left.
  *
- * @param {bigint} lo - The low order 64 bits of the number.
- * @param {bigint} hi - The high order 64 bits of the number.
- * @returns {bigint} The combined 128-bit number represented as a single bigint.
+ * @param  lo - The low order 64 bits of the number.
+ * @param  hi - The high order 64 bits of the number.
+ * @returns  The combined 128-bit number represented as a single bigint.
  */
 export const joinLoHi = (lo = 0n, hi = 0n): bigint => {
```

Comment:
Clarifying my comment above:
```suggestion
export const joinLoHi = (
  /** The low order 64 bits of the number. */
  lo = 0n,
  /** The high order 64 bits of the number. */
  hi = 0n,
): bigint => {
```